content/doc/book/managing/plugins.adoc

```diff
@@ -117,14 +117,19 @@ version of a plugin. In cases where an older release of the plugin is desired,
 a Jenkins administrator can download an older `.hpi` archive and manually
 install that on the Jenkins controller.
 
+Jenkins stores plugins it has downloaded in the `plugins` directory with a `.jpi` suffix, whether the plugins originally had a `.jpi` or an `.hpi` suffix.
+Files installed from the plugin manager or installed through the command line interface will be stored in the `plugins` directory with a `.jpi` suffix.
```

Comment:
Redundant? The first line is a subset of the second? Or how else does Jenkins download plugins than through the plugin manager?